### ISSUE TITLE
Miniclient in Google Play Store is gone

### ISSUE THREAD

**Issue by tsv-kachina** [OpenSageTV/sagetv-miniclient#116]
As title says, Android miniclient doesn't appear to be in Google Play Store anymore. Definitely was there a couple of weeks ago. Did it get purged? Anyone has any additional info on it?

**Comment by Sagetvmaster** [comment]
Yeah, just checked and I don't see it either - hopefully Josh is just arranging an update?

**Comment by Sagetvmaster** [comment]
> As title says, Android miniclient doesn't appear to be in Google Play Store anymore. Definitely was there a couple of weeks ago. Did it get purged? Anyone has any additional info on it?

It seems to still be available in the Amazon app store...

**Comment by DarkWolf7** [comment]
Just went to install it on my new shield and it's definitely gone...

**Comment by bigbillsds** [comment]
Anyone know where the most current .apk file is?  I need to figure out how to sideload it on my googletv boxes.   Kind of shocking that SageTV has been forgotten.  I have spent over a year trying to find a backend with similar functionality.  All of there are lacking most of the things we all take for granted that have been in SagTV since 2006 or so.  Pretty unbelievable.  Was told nextpvr was as good.  Been a few days,   Not great, even missed recording a show already, and I cannot figure out why.  Sad state of affairs.

**Comment by stuckless** [comment]
In the releases section, there's an APK.  I have no idea if it's the most recent.
https://github.com/OpenSageTV/sagetv-miniclient/releases

**Comment by tsv-kachina** [comment]
I use .apk from the link stuckless mentioned, and use adb over the network to sideload it, so no need to rely on questionable file managers.
